src/nanoarrow/nanoarrow.h

```diff
@@ -664,6 +664,10 @@ static inline ArrowErrorCode ArrowBufferAppendBufferView(struct ArrowBuffer* buf
 /// \brief Extract a boolean value from a bitmap
 static inline int8_t ArrowBitGet(const uint8_t* bits, int64_t i);
 
+/// \brief Extract boolean values from a range in a bitmap
+static inline void ArrowBitsGet(const uint8_t* bits, int64_t start_offset, int64_t length,
```

Comment:
What is perhaps a little confusing is that `start_offset` is number of bits for this function, whereas the set functions refer to it as number of bytes. I think that makes sense given the inputs/outputs, but maybe we should change the identifers?

Reply:
That's a much simpler way of expressing this than I was putting out there